autogluon/task/tabular_prediction/predictor.py

```diff
@@ -315,6 +315,42 @@ def fit_summary(self, verbosity=3):
             print("*** End of fit() summary ***")
         return results
 
+    # TODO: Consider adding time_limit option to early stop the feature importance process
+    def feature_importance(self, model=None, dataset=None, features=None, raw=True, silent=False):
+        """
+        Calculates feature importance scores for the given model.
+        Feature scores represent the score reduction the model achieves when the provided data's feature is randomly shuffled among its rows.
+        A score of 0.01 would indicate that the score dropped by 0.01 when the feature was randomly shuffled.
```

Comment:
I don't think we should use "score" to refer to "predictive performance" since it may be confusing as there is feature importance score and predictive performance score.

Reply:
Updated